src/unity/lib/annotation/image_classification.cpp

```diff
@@ -238,7 +250,7 @@ void ImageClassification::_addAnnotationToSFrame(size_t index, int label) {
 
 void ImageClassification::cast_annotations() {
   size_t annotation_column_index = m_data->column_index(m_annotation_column);
-  if(m_data->dtype().at(annotation_column_index) == flex_type_enum::INTEGER) {
+  if (m_data->dtype().at(annotation_column_index) == flex_type_enum::INTEGER) {
```

Comment:
Instead of `m_data->dtype().at(...)`, use `m_data->column_type(m_annotation_column)`